CONTRIBUTE.md

```diff
@@ -14,12 +14,12 @@ After reading this document, you will:
 
 > If you have already installed a supported version of Ansible, you can skip to the [Clone the Repo](#clone-the-repo) step.
 
-This collection requires **Python v3.6 or greater**. If you don't have Python installed, consider using [pyenv](https://github.com/pyenv/pyenv) to install the supported Python runtime.
+This collection requires **Python v3.8 or greater**. If you don't have Python installed, consider using [pyenv](https://github.com/pyenv/pyenv) to install the supported Python runtime.
```

Comment:
There's an inconsistency here between this file and `scripts/run-tests.sh` where we set the min python version to 3.9.  I think we want to fix the `run-tests.sh` file.

We also set `MIN_PYTHON_VERSION: "3.9"` in the GitHub action config 👀

Reply:
According to [ansible-core support matrix](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix), the min python version will be `3.8`.

I'll update eveyrhing to mention only `3.8`

Reply:
I updated it in [this commit](https://github.com/1Password/ansible-onepasswordconnect-collection/pull/83/commits/3c9b99f0879b8863ab9f9cb8eb3a2348589468e5). But let me explain what's going on there

In order to make the tests run all ansible versions in the matrix must be compatible with the python version. Therefore the variable is not MIN_PYTHON_VERSION, but just PYTHON_VERSION which is compatible to run all the tests in the matrix

Reply:
In other words, the minimum python version to run the collection is 3.8, but the version to run sanity tests is 3.9

Reply:
:+1: makes sense. Thank you for clarifying